### PR TITLE
CRM-18752 Fix Custom multi value importer

### PR DESCRIPTION
**Pull request by seamuslee001**
- [CRM-18752: Multi Value Custom Field importer broken on 4.7](https://issues.civicrm.org/jira/browse/CRM-18752)

### CODE REVIEW

File: CRM/Custom/Import/Form/DataSource.php [hunk 1]
```diff
@@ -88,7 +88,9 @@ public function postProcess() {
       'multipleCustomData',
     ));
 
-    $this->submitFileForMapping('CRM_Custom_Import_Parser_Api');
+    $entity = 'multipleCustomData';
+
+    $this->submitFileForMapping('CRM_Custom_Import_Parser_Api', $entity);
```
Comment (colemanw) [inline CRM/Custom/Import/Form/DataSource.php]: nitpick: no need to create a one-off variable here.

**Comment by colemanw** [comment]
Overall code looks good. +1 from me.

**Comment by eileenmcnaughton** [comment]
Alice is testing this & we will merge once she says 'yay'

**Comment by alifrumin** [comment]
@colemanw and @seamuslee001, tested on the jenkins build, looks good to merge to me.

**Comment by eileenmcnaughton** [comment]
Thanks @alifrumin  & @seamuslee001  - merging now!

**Pull request merged**